docs/source/markdown/podman-inspect.1.md

```diff
@@ -1,20 +1,26 @@
 % podman-inspect(1)
 
 ## NAME
-podman\-inspect - Display a container or image's configuration
+podman\-inspect - Display a container or image's configuration.
```

Comment:
The added period is causing a gating test error as this line no longer matches the corresponding line in podman.1.md.

Reply:
I pushed an amended commit which drops the period.